src/plotting.jl

```diff
@@ -1,13 +1,36 @@
 
+####################################################################################################
+####################################### Colors #####################################################
+####################################################################################################
+
+const LIBRARIES = [:Octavian, :MKL, :OpenBLAS, :blis, :Tullio, :Gaius, :Generic];
+"""
+Defines the mapping between libraries and colors
+"""# #0071c5 == Intel Blue
+const COLOR_MAP = Dict(zip(LIBRARIES, distinguishable_colors(length(LIBRARIES), [RGB(1,0,0), colorant"#0071c5"])));
```

Comment:
```suggestion
# make sure colors are distinguishable against white background by adding white to the seed list,
# then deleting it from the resultant palette
palette = distinguishable_colors(length(LIBRARIES) + 1, [colorant"white", RGB(1,0,0), colorant"#0071c5"])
deleteat!(palette, 1)
const COLOR_MAP = Dict(zip(LIBRARIES, palette))
```

Reply:
I swapped `RGB(1, 1, 0)` to [`#66023C`](https://en.wikipedia.org/wiki/Tyrian_purple)

Reply:
Might be a good idea to fix a number instead of `length(LIBRARIES)`, so that if e.g. someone figures out how to add AppleAccelerate the colours don't jump? 

Actually I'm not sure that's enough, ideally omitting one (because something doesn't work on your computer, etc) would also not cause a jump.

Reply:
If you want to omit a library, you can pass a custom library vector to `runbench`, e.g. `runbench(libs = [:Tullio, :MKL])`.

As for adding more, `distinguishable colors` is greedy, and building up the list one at a time.
```julia
julia> using Colors

julia> distinguishable_colors(4, [colorant"white", colorant"#66023C", colorant"#0071c5"])
4-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)

julia> distinguishable_colors(5, [colorant"white", colorant"#66023C", colorant"#0071c5"])
5-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)
 RGB{N0f8}(0.953,0.322,0.0)

julia> distinguishable_colors(6, [colorant"white", colorant"#66023C", colorant"#0071c5"])
6-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)
 RGB{N0f8}(0.953,0.322,0.0)
 RGB{N0f8}(1.0,0.278,0.918)

julia> distinguishable_colors(7, [colorant"white", colorant"#66023C", colorant"#0071c5"])
7-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)
 RGB{N0f8}(0.953,0.322,0.0)
 RGB{N0f8}(1.0,0.278,0.918)
 RGB{N0f8}(0.369,0.78,0.0)

julia> distinguishable_colors(8, [colorant"white", colorant"#66023C", colorant"#0071c5"])
8-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)
 RGB{N0f8}(0.953,0.322,0.0)
 RGB{N0f8}(1.0,0.278,0.918)
 RGB{N0f8}(0.369,0.78,0.0)
 RGB{N0f8}(0.0,0.71,0.729)

julia> distinguishable_colors(9, [colorant"white", colorant"#66023C", colorant"#0071c5"])
9-element Array{RGB{N0f8},1} with eltype RGB{FixedPointNumbers.N0f8}:
 RGB{N0f8}(1.0,1.0,1.0)
 RGB{N0f8}(0.4,0.008,0.235)
 RGB{N0f8}(0.0,0.443,0.773)
 RGB{N0f8}(0.294,0.357,0.0)
 RGB{N0f8}(0.953,0.322,0.0)
 RGB{N0f8}(1.0,0.278,0.918)
 RGB{N0f8}(0.369,0.78,0.0)
 RGB{N0f8}(0.0,0.71,0.729)
 RGB{N0f8}(0.0,0.165,0.184)
```
So as long as they/we add any new libraries to the end of the library list, none of the other colors will change.

Reply:
Also, there is actually no background to these plots produced by Gadfly. My GitHub is in dark mode, so the white line from Tullio was very visible. Gaius was much harder to see.

Reply:
Oh neat, and your axes are mid-gray to work on both. Maybe if just needs to generate colours avoiding both white and black?

Reply:
Good idea, I excluded black as well.